.gitignore

```diff
@@ -113,4 +113,7 @@ packages/dbaeumer*
 .netlify
 
 # Vscode History Extension
-.history
\ No newline at end of file
+.history
+
+# Extension build directory
+packages/**/extension
```

Comment:
under each packages/vscode-* dir an extension directory is created in order to hold just the files necessary for packaging. This was necessary due to using npm workspaces and it attempting to always resolve all deps listed in the package.json file.